packages/indexer-client/src/types/serverTypes.ts

```diff
@@ -360,6 +382,18 @@ export interface IndexerServerMakerStatisticsResponse {
   makers: IndexerServerMaker[];
 }
 
+export interface IndexerServerLeaderboardResponse {
+  positions: IndexerServerLeaderboardPosition[];
+}
+
+export interface IndexerServerLeaderboardRankResponse {
+  position: IndexerServerLeaderboardPosition;
```

Comment:
Doesn't this return `null` when not eligible?

Reply:
Yup, will fix

Reply:
addressed!